.github/workflows/ci.yml

```
@@ -0,0 +1,254 @@
+
+      - name: Generate icons
+        run: yarn icons:generate
+
+      - name: Build icons
+        run: yarn icons:build
+
+      - name: Test icons
+        run: yarn icons:test
+
+      - name: Cache build artifacts
+        uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)